Tools/TestWebKitAPI/Tests/WebKitCocoa/ServiceWorkerBasic.mm

```diff
@@ -1441,7 +1443,10 @@ - (void)userContentController:(WKUserContentController *)userContentController d
     [webView loadRequest:server.request("/second.html"_s)];
     TestWebKitAPI::Util::run(&done);
     done = false;
-    EXPECT_TRUE(retrievedString.contains("\"path\": \"/var/tmp\""_s));
+    String tempDirectoryString = tempDirectory;
+    NSString* tempDirectoryInUse = FileSystem::realPath(tempDirectoryString);
```

Comment:
nit: We can probably call `FileSystem::realPath(tempDirectory)` and get rid of the `tempDirectoryString` local.